tests/test_sessions.py

```diff
@@ -86,15 +89,13 @@ def test_one(worker_id):
 
             span = trace.get_current_span()
             assert span.context.trace_id == 0x1234567890abcdef1234567890abcdef
-            assert span.context.span_id != 0xfedcba0987654321
```

Comment:
I realized these tests for `span_ids` were sort of a freebie; the spans around tests mean that we'd never match the test run or worker-level span anyway.